lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -37,9 +37,22 @@ lh_bg_js="dist/lighthouse-dt-bundle.js"
 lh_worker_dir="$frontend_dir/audits2_worker/lighthouse"
 
 # copy report files
-cp -pPR $report_dir/{report-styles.css,templates.html,renderer} "$fe_lh_dir"
+cp -pPR $report_dir/renderer "$fe_lh_dir"
 echo -e "\033[32m ✓\033[39m Report renderer files copied."
 
 # copy lighthouse-dt-bundle (potentially stale)
 cp -pPR "$lh_bg_js" "$lh_worker_dir/lighthouse-dt-bundle.js"
 echo -e "\033[96m ✓\033[39m (Potentially stale) lighthouse-dt-bundle copied."
+
+# bundle html generator
+node -e "
+  const htmlReportAssets = require('./lighthouse-core/report/html/html-report-assets.js');
+  const reportGenerator = require('./lighthouse-core/report/report-generator.js');
+  console.log('const htmlReportAssets =', JSON.stringify(htmlReportAssets, null, 2), ';');
+  console.log('class ReportGenerator {');
+  console.log('  static Assets = htmlReportAssets;');
+  console.log('  static', reportGenerator.generateReportHtml.toString());
```

Comment:
this feels....fragile to say the least :)

Reply:
What about bundling w/ browserify + this plugin? https://github.com/browserify/brfs

The file system access is what made it tricky to consume in DT.

Reply:
Not doing above, but I did a refactor. Changed the report generator module to load from Runtime.cachedAssets if the env is Devtools, and replaced the node bundling code with just copying those assets.

Reply:
Bonus - fewer bytes wasted from string escaping :)